docker-compose.yml

```diff
@@ -1,16 +1,25 @@
-version: '3.1'
+version: "3.5"
 services:
-
-  hub-frontend-redis:
+  frontend:
+    image: quay.io/hmpps/prisoner-content-hub-frontend
+    build:
+      context: .
+    env_file:
+      - prisoner-content-hub-frontend.env
+    ports:
+      - 3000:3000
+    networks:
+      - prisoner_content_hub
+  frontend-redis:
     image: 'bitnami/redis:5.0'
     networks:
-      - hmpps
+      - prisoner_content_hub
     container_name: hub-frontend-redis
     environment:
       - ALLOW_EMPTY_PASSWORD=yes
     ports:
-      - '6379:6379'
+      - '6378:6378'
```

Comment:
I've had to update the port number here as 6379 is already in use for the backend Redis instance.